sockeye/data_io.py

```diff
@@ -1432,11 +1432,11 @@ def __init__(self,
                  target_data_name=C.TARGET_NAME,
                  label_name=C.TARGET_LABEL_NAME,
                  num_factors: int = 1,
-                 no_shuffle: Optional[bool] = False,
+                 shuffle: Optional[bool] = False,
```

Comment:
I think the default should be True here as well. Also, what about the ShardedIter?

Reply:
Oh yes, I meant to set it to True there as well. For the sharded iter, is it enough to just also supply the `shuffle` argument to the underlying iterator `self.shard_iter`? The shard files (`self.shards_fnames`) are also shuffled currently.